api/chat.go

```diff
@@ -6,81 +6,72 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/joschahenningsen/TUM-Live/dao"
-	"github.com/joschahenningsen/TUM-Live/model"
-	"github.com/joschahenningsen/TUM-Live/tools"
 	"net/http"
 	"strconv"
 	"time"
 
-	"github.com/gabstv/melody"
+	"github.com/joschahenningsen/TUM-Live/dao"
+	"github.com/joschahenningsen/TUM-Live/model"
+	"github.com/joschahenningsen/TUM-Live/tools"
+	"github.com/joschahenningsen/TUM-Live/tools/realtime"
+
 	"github.com/getsentry/sentry-go"
 	"github.com/gin-gonic/gin"
 	log "github.com/sirupsen/logrus"
 )
 
-var m *melody.Melody
-
-const maxParticipants = 10000
-
-func configGinChatRouter(router *gin.RouterGroup, daoWrapper dao.DaoWrapper) {
-	routes := chatRoutes{daoWrapper}
-
-	wsGroup := router.Group("/:streamID")
-	wsGroup.Use(tools.InitStream(daoWrapper))
-	wsGroup.GET("/messages", routes.getMessages)
-	wsGroup.GET("/active-poll", routes.getActivePoll)
-	wsGroup.GET("/users", routes.getUsers)
-	wsGroup.GET("/ws", routes.ChatStream)
-	if m == nil {
-		log.Printf("creating melody")
-		m = melody.New()
-	}
+const (
+	ChatRoomName = "chat/:streamID"
+)
 
-	m.HandleConnect(connHandler)
+var routes chatRoutes
+
+//var m *melody.Melody
```

Comment:
I guess we don't need that out-commented code anymore. Therefore, could you remove it please?

Reply:
mac ... 👍